### ISSUE TITLE
v2.0 NPM release does not contain any code

### ISSUE THREAD

**Issue by rettgerst** [peerigon/extract-loader#37]
```
$ ls node_modules/extract-loader
CHANGELOG.md LICENSE      README.md    package.json
```

**Comment by jperocho** [comment]
Same problem when updated my npm packages.

**Comment by vladimiry** [comment]
I confirm that, main script is pointed to nonexistent file `"main": "lib/extractLoader.js",`.

**Comment by benurb** [comment]
Thanks for the report. This should be fixed with v2.0.1
Sorry for the inconvenience 😞 

```
node_modules/extract-loader/LICENSE
node_modules/extract-loader/CHANGELOG.md
node_modules/extract-loader/README.md
node_modules/extract-loader/package.json
node_modules/extract-loader/lib/extractLoader.js
```
